### PR TITLE
[fronius] Add the new channels to the documentation

### PR DESCRIPTION
**Pull request by jimtng**
Add the documentation for the channels that were added in #10757

### CODE REVIEW

File: bundles/org.openhab.binding.fronius/README.md [hunk 1]
```diff
@@ -50,30 +49,33 @@ The binding has no configuration options, all configuration is done at `bridge`,
...
+| `inverterdatachannelidc`             | Number               | DC current                                                                                                        |
+| `inverterdatachanneluac`             | Number               | AC voltage                                                                                                        |
+| `inverterdatachanneludc`             | Number               | DC voltage                                                                                                        |
+| `inverterdatadevicestatuserrorcode`  | Number               | Device error code                                                                                                 |
+| `inverterdatadevicestatusstatuscode` | Number               | Device status code<br />`0` - `6` Startup<br />`7` Running <br />`8` Standby<br />`9` Bootloading<br />`10` Error |
+| `powerflowchannelpgrid`              | Number               | Power (+ from grid, - to grid)                                                                                    |
```
Comment (lolodomo) [inline bundles/org.openhab.binding.fronius/README.md:73]: Type is Number:Power
Please check the example.

File: bundles/org.openhab.binding.fronius/README.md [hunk 2]
```diff
@@ -50,30 +49,33 @@ The binding has no configuration options, all configuration is done at `bridge`,
...
+| `powerflowchannelpload`              | Number               | Power (+ generator, - consumer)                                                                                   |
```
Comment (lolodomo) [inline bundles/org.openhab.binding.fronius/README.md:74]: Type is Number:Power
Please check the example.

File: bundles/org.openhab.binding.fronius/README.md [hunk 3]
```diff
@@ -50,30 +49,33 @@ The binding has no configuration options, all configuration is done at `bridge`,
...
+| `powerflowchannelpakku`              | Number               | Power (+ charge, - discharge)                                                                                     |
```
Comment (lolodomo) [inline bundles/org.openhab.binding.fronius/README.md:75]: Type is Number:Power
Please check the example.

File: bundles/org.openhab.binding.fronius/README.md [hunk 4]
```diff
@@ -156,6 +158,9 @@ Number StatusCode { channel="fronius:powerinverter:mybridge:myinverter:inverterd
 Number Grid_Power { channel="fronius:powerinverter:mybridge:myinverter:powerflowchannelpgrid" }
 Number Load_Power { channel="fronius:powerinverter:mybridge:myinverter:powerflowchannelpload" }
 Number Battery_Power { channel="fronius:powerinverter:mybridge:myinverter:powerflowchannelpakku" }
+Number:Power Production_Power { channel="fronius:powerinverter:mybridge:myinverter:powerflowchannelppv" }
+Number:Power Inverter1_Power { channel="fronius:powerinverter:mybridge:myinverter:powerflowinverter1power" }
+Number Inverter1_SOC { channel="fronius:powerinverter:mybridge:myinverter:powerflowinverter1soc" }
```
Comment (lolodomo) [inline bundles/org.openhab.binding.fronius/README.md:163]: Number:Dimensionless

**Review by lolodomo — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.fronius/README.md (hunk 1 above)
Comment (jimtng) [reply bundles/org.openhab.binding.fronius/README.md]: Thanks for picking this up

File: bundles/org.openhab.binding.fronius/README.md (hunk 3 above)
Comment (jimtng) [reply bundles/org.openhab.binding.fronius/README.md]: Thanks for picking this up!

File: bundles/org.openhab.binding.fronius/README.md (hunk 4 above)
Comment (jimtng) [reply bundles/org.openhab.binding.fronius/README.md]: Thanks for picking this up

**Comment by jimtng** [comment]
Added the missing dimensions

**Review by lolodomo — APPROVED** [review]
LGTM

**Pull request merged**